app/controllers/TimeController.scala

```diff
@@ -104,8 +105,8 @@ class TimeController @Inject()(userService: UserService,
     for {
       userJs <- userService.compactWrites(user)
       timeJs <- timeSpanDAO.findAllByUserWithTask(user._id,
-                                                  Some(startDate.getTimeInMillis),
-                                                  Some(endDate.getTimeInMillis))
+                                                  Some(Instant(startDate.getTimeInMillis)),
+                                                  Some(Instant(endDate.getTimeInMillis)))
```

Comment:
Maybe add another conversion to `Instant` itself, similar to the other ones that are already in place?